### DISCUSSION TITLE
Can I contribute to creating samples for Prism for .NET MAUI?

### DISCUSSION THREAD

**Discussion by niimima** [PrismLibrary/Prism#2955] [.NET MAUI] [answered]
Implementing or modifying the Prism library itself is challenging for me, but I believe I can contribute to creating samples.
It seems there is no sample repository yet, but will it be created at some point in the future?
If a repository is created, I would appreciate it if guidelines on what should be created or any other instructions could be provided.

**Comment by dansiegel** [comment, marked as answer]
We would certainly welcome the help in getting the samples put together. There is a little bit of housekeeping we need to do in setting up the repo and providing a sample-template project similar to what we have for Forms. But once I get that up certainly feel free.

**Comment by giuseppenovielli** [comment]
@niimima Here there are some samples.
https://github.com/giuseppenovielli/MAUI_XF-Prism-Full-Navigation-Example
